models/modelling/olids/risk_stratification/int_efi2_chronology.sql

```
@@ -0,0 +1,421 @@
+    ),
+
+    -- Full scored population (one row per person), sourced from the patient_list
+    -- spine, NOT int_efi2_cohort_snomed_codes. The cohort table is an inner join to
+    -- the codelist, so it only contains persons with >= 1 eFI2 code. The paper
+    -- assigns a "missing" lifestyle category (BMI, alcohol) so that patients with no
+    -- lifestyle data still receive a score, and evaluates polypharmacy across the
+    -- whole population. Drawing from the cohort table instead would deny the
+    -- missing-lifestyle weights and polypharmacy to code-sparse persons, understating
+    -- their score. Patient_list is already the correct denominator (living, age >= 65).
+    person_unique as (
+        select distinct person_id, end_date from {{ ref("int_efi2_patient_list") }}
```

SQLFluff (4.2.2)

[error] Select targets should be on a new line unless there is only one select target.

(LT09)